packages/webpack-config/webpack/webpack.common.js

```diff
@@ -299,7 +299,7 @@ module.exports = function(env = {}, argv) {
         new ModuleScopePlugin(locations.template.folder, [locations.packageJson]),
       ],
       symlinks: false,
-      extensions: ['.web.ts', '.ts', '.tsx', '.web.js', '.js', '.jsx', '.json'],
+      extensions: ['.web.ts', 'web.tsx', '.ts', '.tsx', '.web.js', 'web.jsx', '.js', '.jsx', '.json'],
```

Comment:
Looks like it should be `'.web.tsx'` and not `'web.tsx'`?

Reply:
Yeah, you're right. I made sure to double check it and still missed it 😂

Reply:
Fixed by #538